frontend/package.json

```diff
@@ -101,23 +101,28 @@
     ],
     "development": [
       "last 1 chrome version",
-      "last 1 firefox version",
-      "last 1 safari version"
```

Comment:
OK. That was from when I was intending to also update Rapid to v2 in this PR. I ended up dropping that, since I was having some issues with integration -- there are some race conditions that I needed to work around or fix upstream.

Anyway, the specific error I was getting was
> SyntaxError: frontend/node_modules/RapiD/dist/rapid.min.js: When using '@babel/plugin-transform-parameters', it's not possible to compile `super()` in an arrow function with default or rest parameters without compiling classes.

The upstream ticket that led me to drop Safari was https://github.com/pixijs/pixijs/issues/8733.

With that said, there is a fix in newer versions of pixijs, which Rapid updated to after I had this issue, so I can probably readd it.